src/modules/models/HeatLossArrow/HeatLossArrow.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+Auto-generated by: https://github.com/pmndrs/gltfjsx
+Command: npx gltfjsx@6.5.2 OriginalArrow.glb --transform --types 
+Files: OriginalArrow.glb [21.84KB] > Arrow.glb [3.86KB] (82%)
+Author: Alihan (https://sketchfab.com/Dare0)
+License: CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)
+Source: https://sketchfab.com/3d-models/direction-arrow-6ef46718c7b242e39fcad7f27ee858a5
+Title: Direction Arrow
+*/
+import { useRef } from 'react';
+
+import { useGLTF } from '@react-three/drei';
+import { useFrame } from '@react-three/fiber';
+import { Color, Group, Mesh, MeshStandardMaterial, Vector3 } from 'three';
+import { GLTF } from 'three-stdlib';
+
+import { MODELS_3D_ROOT_PATH } from '@/config/models';
+import { Position } from '@/types/wall';
+import { formatHeatLossRate, powerConversionFactors } from '@/utils/heatLoss';
+
+import { HeatLossTextArrow } from './HeatLossTextArrow';
+
+const GLB_FILE_PATH = `${MODELS_3D_ROOT_PATH}/HeatLossArrow.glb`;
```

Comment:
For this maybe you can use the url import feature of vite ? 
```suggestion
import GLB_FILE_PATH from '../models/HeatLossArrow.glb?url';
```

For reference: https://v2.vitejs.dev/guide/assets.html#explicit-url-imports